File: 1.FDP/3° Competencia/Java/Repeticion/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		for (int i=0; i<3; i++) {
			System.out.println("Calcular el area de un triangulo.\n");
			double Base, Altura, Area;
			System.out.println("Base: ");
			Base= scan.nextDouble();
			if (Base > 0) {
				System.out.println("Altura: ");
				Altura= scan.nextDouble();
				if (Altura > 0) {
						Area= (Base*Altura)/2;
					System.out.println("Area= " + Area);
					System.out.println("///////////////////////////.");
				} else {
					System.out.println("Error en la altura");
					System.out.println("//////////////////////////");
				}
			} else {
				System.out.println("Error en la base");
				System.out.println("//////////////////////////");
			}
		}
	}
}

File: 1.FDP/2° Competencia/C++/Repeticion/Ejercicio3.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Calcular el area de un triangulo.\n";
	double Altura, Base, Area;
	int respuesta;
	cout << "Cuantas veces deceas que se repita: ";
	cin >> respuesta;
	for ( int i=0; i<respuesta; i++) {
		cout << "Ingresar la base:";
		cin >> Base;
		if (Base > 0){
			cout << "Ingresar la altura:";
			cin >> Altura;
			if (Altura > 0) {
				Area= (Altura*Base)/2;
				cout << "Area = " << Area << "\n";
				cout << "/////////////////////////////.\n";
			} else {
				cout << "Error en la altura.\n";
				cout << "///////////////////////.\n";
			}
		} else {
			cout << "Error en la base.\n";
			cout << "///////////////////////.\n";
		}
	}		
	
	return 0;
}